doc/nrf/releases/release-notes-latest.rst

```diff
@@ -197,34 +321,108 @@ The following list summarizes the most important changes inherited from upstream
   * Fixed a possible NULL pointer dereference when using any of the ``fs_`` functions.
     The functions will now return an error code in this case.
   * Fixed a garbage-collection issue in the NVS subsystem.
+  * Added the Kconfig option :option:`CONFIG_FS_FATFS_EXFAT` for enabling exFAT support.
+  * Added support for file open flags to fs and POSIX API.
+
+* Management:
+
+  * MCUmgr:
+
+    * Moved mcumgr into its own directory.
+    * Switched UDP port to use the kernel stack.
+    * Added missing socket close in error path for SMP.
+
+  * Added support for Open Supervised Device Protocol (OSDP) (see :option:`CONFIG_OSDP`).
+
+  * updatehub:
+
+    * Moved updatehub from :file:`lib` to :file:`subsys/mgmt`.
+    * Fixed out-of-bounds access and added a return value check for ``flash_img_init()``.
+    * Fixed a ``getaddrinfo`` resource leak.
+
+* Settings:
+
+  * Updated the implementation to return an error rather than faulting if there is an attempt to read a setting from a channel that does not support reading.
+  * Disallowed modifying the content of a static subtree name.
+
+* LVGL:
+
+  * Updated the library to the new major release v7.0.2.
+  * Aligned LVGL Kconfig constants with suggested defaults from upstream.
+
+* Tracing:
+
+  * Updated the API to check if the init function exists prior to calling it.
```

Comment:
I changed "tracing backed API" to just "API" here - is that okay?

Reply:
I think I've added this. Fair enough "API" seems better.